project-fob/Controllers/HomeController.cs

```diff
@@ -97,69 +104,96 @@ public ActionResult meetingPageUser(string meetingId, string password)
                 {
                     password = "";
                 }
-                Meeting meet = db.Meeting.SingleOrDefault(m => m.MeetingId == meetingId && m.Active);
+                //Meeting meet = db.Meeting.SingleOrDefault(m => m.MeetingId == meetingId && m.Active);
```

Comment:
You might as well just delete code instead of commenting it out, the history is in git so we can always get it back. If you let commented out code hang around for too long it just gets in the way when reading the code.